### DISCUSSION TITLE
Project Presentation 12/04/2024

### DISCUSSION THREAD

**Discussion by umarbilgrammi** [ecn310/course-project-taxes-tariffs#22] [Announcements]
New working/null hypothesis since we also started looking at developed countries: 

Working Hypothesis: If consumption taxes like VAT increases in a developed or developing country, then tariffs decrease.
Null hypothesis: Increases in consumption taxes like VAT have no effect on tariffs in developed or developing countries

**Comment by DylanTh20** [comment]
![HistogramGDPVAT198501989](https://github.com/user-attachments/assets/d317d8c3-f4e4-401f-9007-4b726a0270c8)

**Comment by DylanTh20** [comment]
![HistogramGDPVAT2017-2022](https://github.com/user-attachments/assets/a2c0c01a-c870-4e70-9017-dded75ba6e9c)

**Comment by DylanTh20** [comment]
![ScatterplotGdpVatGreenGoblin](https://github.com/user-attachments/assets/4c1088aa-7b17-4674-9556-0452bb527df6)

**Comment by kbuzard** [comment]
> Null hypothesis: In a developed or developing country, there is no difference in tarrifs when value added tax changes.

@ecn310/taxes-tariffs The usually way economists would phrase this is something like "Increases in Value Added Taxes (VAT) have no effect on tariffs."

**Comment by Ksarrge** [comment]
@kbuzard here is our presentation

**Comment by Ksarrge** [comment]
**Comments from Presentation:**
- specify GDP per capital
- VAT = ___ percent of revenue
- put legend in space in graph
- want the correlation and significance between our different scatter plots
- bottom line in turns of null = rejected 
- gdp = independent variable put it on the x-axis
- add a marker for GDP cut off
- change bins in bar graphs
- no countries gdp per capita above 40,000 but scatterplot showed in 50,000 = tell if cut off some of the results
- Make 2 two-way histograms, one for developed and one for developing
- check histogram by doing a one-way histogram for both gdp and vat and then compare to the two-way histogram
- about 7% of the observations are in the smallest bin
- only had about 18% of data in bar graph, what happened to the other ~80% -> frequency of observations data that is not included in specific years, scale graph so it can add up to 100%. EX: if 20% of observations divide everything by 5 to scale everything to 100%
- add line of best fit in scatterplot to see correlation
